python/hail/dataset.py

```diff
@@ -3238,18 +3241,26 @@ def lmmreg(self, kinshipMatrix, y, covariates=[], global_root="global.lmmreg", v
 
         **Kinship Matrix**
 
-        FastLMM uses the Realized Relationship Matrix (RRM) for kinship. This can be computed with :py:meth:`~hail.VariantDataset.rrm`. However, any instance of :py:class:`KinshipMatrix` may be used, so long as ``sample_list`` contains the complete samples of the caller variant dataset in the same order.
+        FastLMM uses the Realized Relationship Matrix (RRM) for kinship. This can be computed with :py:meth:`~hail.VariantDataset.rrm`. However, any instance of :py:class:`KinshipMatrix` may be used, so long as ``sample_list`` contains the complete samples of the caller variant dataset in the same order. Currently, due to the fact that the eigendecomposition is done locally, this matrix (or the LD matrix below) can only be a maximum size of 32,000 X 32,000.
+
+        **LD Matrix**
+
+        The rank of the kinship matrix (RRM) resulting from a genetic matrix on :math:`n` samples and :math:`m` variants is at most the minimum of :math:`n` and :math:`m`. So to improve efficiency when :math:`m` is less than :math:`n`, this method alternatively accepts an LD matrix, the eigenvectors of which are then transformed to the first :math:`m` eigenvectors of the RRM using the relationship between left and right singular vectors as defined by the singular value decomposition of the normalized genetic matrix. Passing in the RRM or LD matrix will result in mathematically identical statistics. The n_eigs and dropped_variance_fraction parameters may be used in addition to further improve performance using approximate rather than exact inference (see next section).
```

Comment:
Can we collapse the LD Matrix and Kinship Matrix sections into a "gram matrix" section and just note that one or the other is required and due to technical reasons they must be less than 32k x 32k?